common/src/main/java/tds/common/web/advice/ExceptionAdvice.java

```diff
@@ -52,6 +58,8 @@
     @ResponseStatus(value = HttpStatus.BAD_REQUEST)
     @ResponseBody
     ResponseEntity<ExceptionMessageResource> handleInvalidParametersException(final IllegalArgumentException ex) {
+        LOG.info("Invalid request error", ex);
```

Comment:
After an offline discussion bumped this to warn since IAE can be thrown for other reasons that bad data from the client.